src/Employer/Employer.Web/wwwroot/javascripts/application.js

```diff
@@ -56,37 +56,6 @@ sfa.navigation = {
     }
 };
 
-sfa.hookupExampleVacancyToggle = function (showExampleText, hideExampleText, showExampleVacancyCookieName) {
-    var $exampleLink = $("#example_link"),
-        $exampleVacancy = $("#example_vacancy"),
-        $exampleVacancyContainer = $("#example-vacancy-js-container");
-
-    $exampleVacancyContainer.show();
-
-    var showExampleVacancyCookie = Cookies.get(showExampleVacancyCookieName);
-
-    if (typeof (showExampleVacancyCookie) === "undefined") {
-        $exampleVacancy.hide();
-    }
-    else {
-        $exampleLink[0].innerText = hideExampleText;
-    }
-
-    $exampleLink.click(function () {
-        if ($exampleVacancy.is(":visible")) {
-            Cookies.remove(showExampleVacancyCookieName);
```

Comment:
This was making use of a JS cookie library. Should we remove that as well.

Reply:
👍 I'll remove this library